packages/babel-plugin-ima-logger/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const base = require("../../jest.config.base.js");
+
+module.exports = {
+    ...base,
+    globals: {

```

Comment:
Coverage should be defined in the root only.

Reply:
Every package has own coverage values right now. If this settings will be moved into root config only then all packages will have same coverage. Do we want this solution?

Reply:
From the coverage output, it seems that the coverage is collected and evaluated from all packages at once (if we run tests from the root), so it makes sense to have single coverage definition.